tests/cupyx_tests/scipy_tests/special_tests/test_log_softmax.py

```diff
@@ -0,0 +1,83 @@
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+import scipy.special  # NOQA
+
+
+atol = {'default': 1e-6, cupy.float64: 1e-14}
+rtol = {'default': 1e-6, cupy.float64: 1e-14}
+
+atol_low = {'default': 1e-6, cupy.float16: 1e-3, cupy.float64: 1e-14}
+rtol_low = {'default': 1e-6, cupy.float16: 1e-3, cupy.float64: 1e-14}
+
+
+@testing.with_requires('scipy')
+class TestLogSoftmax:
+
+    @testing.for_dtypes('bhilefdFD')
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
+    def test_log_softmax_ndarray_1(self, xp, scp, dtype):
+        a = testing.shaped_random((40, 50), xp, dtype=dtype)
+        return scp.special.log_softmax(a)
```

Comment:
```suggestion
    @testing.for_all_dtypes(no_bool=True)
    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
    def test_log_softmax_ndarray_1(self, xp, scp, dtype):
        if xp.dtype(dtype).kind == 'u':
            # Unsigned integers make underflows in numpy (eventually seen as
            # overflows in `np.exp(tmp)`)
            pytest.skip()
        a = testing.shaped_random((40, 50), xp, dtype=dtype)
        return scp.special.log_softmax(a)```

Reply:
ah, I recalled that `x - x_max` underflows with unsigned integers, and eventually makes overflows in `np.exp(tmp)`. Let's skip tests for unsigned ints.

```
    tmp = x - x_max
    exp_tmp = np.exp(tmp)
```

Reply:
Skipped the tests. Thanks for the inputs :)